vendor/github.com/SmartBFT-Go/consensus/pkg/types/types.go

```diff
@@ -111,9 +111,8 @@ type Reconfig struct {
 }
 
 type SyncResponse struct {
-	Latest     Decision
-	Reconfig   ReconfigSync
-	RequestDel []RequestInfo
```

Comment:
I never liked the `RequestDel` thing. It does not help since when you sync from another node, it only contains the requests in the last block and not in all blocks that were synced.

Reply:
I didn't like it either. So when your solution appeared, I deleted it.
But, as I remember, it was accumulating requests from all blocks.